src/scanner.c

```diff
@@ -151,7 +151,7 @@ bool tree_sitter_rescript_external_scanner_scan(
     const bool* valid_symbols
     ) {
   ScannerState* state = (ScannerState*)payload;
-  const in_string = state->in_quotes || state->in_backticks;
+  bool const in_string = state->in_quotes || state->in_backticks;
```

Comment:
```suggestion
  bool in_string = state->in_quotes || state->in_backticks;
```

This was basically a typo. An idiomatic way to express what happens here is just using a regular `bool` variable. Are you OK with it?